components/Grid/Grid.styles.ts

```diff
@@ -50,6 +50,8 @@ export const gridNumCols = {
   },
 };
 
+export const rtl = '[direction:rtl] children:[direction:ltr]';
```

Comment:
This trick makes the grid row items fill from right to left, but need to change the direction of the children from left to right. It's too bad we don't have a CSS property that's kind of like flex row-reverse.